charts/k8tz/templates/controller.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "k8tz.selectorLabels" . | nindent 8 }}
+        {{- include "k8tz.labels" . | nindent 8 }}
```

Comment:
pod labels are different than other resources. for example, the label `app.kubernetes.io/managed-by` that exists in `k8tz.labels` doesn't fit to the pods, because they managed by the Deployment and not directly by Helm.

what I meant is that i prefer to have macro named `k8tz.podLabels` that consist of `k8tz.selectorLabels` plus `.Values.labels` and use it here.